docs/standard/microservices-architecture/docker-application-development-process/docker-app-development-workflow.md

```diff
@@ -1,216 +1,339 @@
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 09/05/2018
+ms.date: 09/27/2018
 ---
 # Development workflow for Docker apps
 
-The application development life cycle starts at each developer’s machine, where the developer codes the application using their preferred language and tests it locally. No matter which language, framework, and platform the developer chooses, with this workflow, the developer is always developing and testing Docker containers, but doing so locally.
+The application development life cycle starts at each developer's machine, where the developer codes the application using their preferred language and tests it locally. No matter which language, framework, and platform the developer chooses, with this workflow, the developer is always developing and testing Docker containers, but doing so locally.
 
 Each container (an instance of a Docker image) includes the following components:
 
-- An operating system selection (for example, a Linux distribution, Windows Nano Server, or Windows Server Core).
+-   An operating system selection (for example, a Linux distribution, Windows Nano Server, or Windows Server Core).
 
-- Files added by the developer (application binaries, etc.).
+-   Files added by the developer (application binaries, etc.).
 
-- Configuration information (environment settings and dependencies).
+-   Configuration information (environment settings and dependencies).
 
 ## Workflow for developing Docker container-based applications
 
-This section describes the *inner-loop* development workflow for Docker container-based applications. The inner-loop workflow means it is not taking into account the broader DevOps workflow and just focuses on the development work done on the developer’s computer. The initial steps to set up the environment are not included, since those are done only once.
+This section describes the *inner-loop* development workflow for Docker container-based applications. The inner-loop workflow means it is not considering the broader DevOps workflow (that can include up to production deployment) and just focuses on the development work done on the developer's computer. The initial steps to set up the environment are not included, since those are done only once.
 
 An application is composed of your own services plus additional libraries (dependencies). The following are the basic steps you usually take when building a Docker application, as illustrated in Figure 5-1.
 
-![Step-by-step workflow for developing Docker containerized apps graphic](./media/image1.png)
+![The development process for Docker apps: 1 - Code your App, 2 - Write Dockerfile/s, 3 - Create images defined at Dockerfile/s, 4 - (optional) Compose services in the docker-compose.yml file, 5 - Run container or docker-compose app, 6 - Test your app or microservices, 7 - Push to repo and repeat. ](./media/image1.png)
```

Comment:
do you have access to our internal sites? Here's the guidance for file names:
https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-create-screenshot?branch=master#file-naming-tips

But here's the important part:
"Save the image files into the media folder adjacent to the Markdown file where they are rendered, in a subfolder matching the article name. For example, /media/article-name/description.png."

The image file name should also be descriptive because they help with SEO as well. 

I think the previous alt-text was better. It should be a sentence.

Reply:
Kept the alt as a description.